src/main/java/io/xpring/xpring/XpringClient.java

```diff
@@ -0,0 +1,60 @@
+package io.xpring.xpring;
+
+import io.xpring.payid.PayIDClientInterface;
+import io.xpring.payid.PayIDException;
+import io.xpring.xrpl.Wallet;
+import io.xpring.xrpl.XRPClientInterface;
+import io.xpring.xrpl.XRPException;
+
+import java.math.BigInteger;
+
+/**
+ * Composes interactions of Xpring services.
+ */
+public class XpringClient {
+  /**
+   * A {@link PayIDClientInterface} used to interact with the Pay ID protocol.
+   */
+  @SuppressWarnings("checkstyle:AbbreviationAsWordInName")
+  private PayIDClientInterface payIDClient;
```

Comment:
Not sure if this is generated, but if not it should idiomatically be called `PayIdClient`
```suggestion
  private PayIDClient payIdClient;
```